modules/common/src/test/java/com/ibm/cloud/cloudant/SdkCommonTest.java

```diff
@@ -39,4 +40,11 @@ public void testGetSdkHeaders() {
         String expectedUserAgentRegex = String.format("%s/%s .*", SdkCommon.getProjectName(), SdkCommon.getVersion());
         assertTrue(userAgent.matches(expectedUserAgentRegex));
     }
+
+    @Test
+    public void testAnalyticsHeaders() {
+        Map<String, String> headers = SdkCommon.getSdkHeaders("service1", "v1", "operation1");
+        String userAgent = headers.get("X-IBMCloud-SDK-Analytics");
```

Comment:
nit: rename the var, it's a little confusing calling this `userAgent`

Reply:
Done in ce72d3d